### DISCUSSION TITLE
Sverchok for Blender 4.3.2- Freecad library for Blender 4.2.2

### DISCUSSION THREAD

**Discussion by lcbt97** [nortikin/sverchok#5180] [Q&A] [answered]
Windows 11
Blender 4.2.2 and 4.3.2
Sorry for boring you, but I can't find any solution in previous Q&A or on any forum/site.
I installed Blender 4.2.2 with Python 3.11.7, and the only extra node I can't install is the Freecad library. The path says the package is not available.
![image](https://github.com/user-attachments/assets/24327780-31c2-4a55-9f97-db2d07f45a98)

I already tried to follow the installation guide and the https://wiki.freecad.org/Blender_Sverchok_Addon_Setup.

Then, I installed Blender 4.3.2 with Freecad 1.0 RC. They use the same version of Python (3.11.9), but I have trouble starting Sverchok, and I can't check if that combination works.
![image](https://github.com/user-attachments/assets/32fdc4ee-af45-4bd8-9748-4caf9ed5f17b)

Could you please help me as soon as possible? I have limited time to finish my thesis, and without the use of solid geometries with Sverchok, I am completely stuck. Thank you so much!

**Comment by rendetto** [comment, marked as answer]
Create(if not present) a file called **freecad_path.pth** in your Blender Python "site-packages" folder e.g. D:\blender-4.2.2-windows-x64\4.3\python\lib\site-packages
The **freecad_path.pth** file should contain only the path to your FreeCAD bin folder e.g. D:\FreeCAD\FreeCAD_1.0.0RC2-conda-Windows-x86_64-py311\bin (no need for quotes or other syntax)
Save **freecad_path.pth** file and restart Blender if it's running. You can clear the field in Sverchok UI pointing to the FreeCAD bin folder before restarting Blender.

**Notes:**
After FreeCAD 1.0 **RC2** there are some changes that are braking the compatibility (see https://forum.freecad.org/viewtopic.php?p=796546&hilit=sverchok&sid=7f5dfcdd3d8249b422a8c8ed88bdf96d#p796546 and https://github.com/nortikin/sverchok/discussions/5174). 

So you need to use [https://github.com/FreeCAD/FreeCAD-Bundle/releases/tag/1.0rc2](url) as it's the last working compatible version.

The conda package you create uses precompiled FreeCAD from the latest code (if you didn't specified something else) so... it's newer than RC2 and therefore is not compatible with Sverchok/Blender (at least for now).

Good luck with your thesis!

**Reply by lcbt97** [reply]
Thank you so much! you save my thesis!
